autoscale/src/main/java/com/sequenceiq/periscope/service/ClusterService.java

```diff
@@ -272,6 +272,12 @@ public List<Long> findClusterIdsByStackTypeAndPeriscopeNodeIdAndAutoscalingEnabl
         return clusterRepository.findClusterIdsByStackTypeAndPeriscopeNodeIdAndAutoscalingEnabled(stackType, nodeId, autoscalingEnabled);
     }
 
+    public List<Long> findClusterIdsByStackTypeAndPeriscopeNodeIdAndAutoscalingEnabledAndScalingTriggers(StackType stacktype, String periscopeNodeId,
```

Comment:
Commented elsewhere also: Very confused about what this query is doing.

Also - do we have to make changes to the ClusterService as part of this patch. Can the Status events be completely de-linked from regular Cluster operations ?